.github/workflows/verify.yml

```diff
@@ -16,12 +16,13 @@ jobs:
   branch:
     runs-on: ubuntu-latest
     outputs:
-      branch_name: ${{ steps.branch.outputs.branch_name }}"
+      branch_name: ${{ steps.branch.outputs.branch_name }}
     steps:
       - id: branch
+        name: Evaluate branch name and set as output
         run: |
-          BRANCH_REF=${GITHUB_HEAD_REF:-$GITHUB_REF}
-          BRANCH_NAME=${BRANCH_REF##*\/}
+          BRANCH_NAME=${GITHUB_HEAD_REF:-$GITHUB_REF_NAME}
```

Comment:
I don't think this is correct, as _REF have refs/heads/ before while NAME only has the branch name, sadly there is no GITHUB_HEAD_REF_NAME var yet